source/docs/basics-of-scripting/memory-flags-and-definitions.rst

```diff
@@ -220,7 +220,72 @@ Of course, you can do a bit more with flags than you can with definitions.
 2B. Math With Flags
 ~~~~~~~~~~~~~~~~~~~
 
+You can do more than just store raw information with flags - you can also do math! Flag math operations support addition, subtraction, multiplication, and division. The operation you specify will be performed on the value currently stored. Note that this only works for flags with numerical values - if you have stored text on a flag, you cannot use math to change the value!
```

Comment:
This line doesn't seem to obey the 120-visible-character limit. Of course it doesn't matter to the frontend user, but for backend users who will be seeing the source and editing it (hopefully), they need to be able to read the backend without having to scroll horizontally.